### ISSUE TITLE
Consider supporting CoNLL-U Plus

### ISSUE THREAD

**Issue by EmilStenstrom** [EmilStenstrom/conllu#33]
More info available here: https://universaldependencies.org/ext-format.html

**Comment by EmilStenstrom** [comment]
The CoNLL-U Plus format has column definitions as metadata:
```
# global.columns = ID FORM LEMMA UPOS XPOS FEATS HEAD DEPREL DEPS MISC
```

Could we use these fields to parse non-standard sentences without having to specify any fields when parsing?

**Comment by EmilStenstrom** [comment]
Fixed in 1f460a8
